openmp/runtime/src/kmp_settings.cpp

```diff
@@ -4988,9 +4985,6 @@ static void __kmp_stg_parse_adaptive_lock_props(const char *name,
       if (num < 0) { // The number of retries should be >= 0
         msg = KMP_I18N_STR(ValueTooSmall);
         num = 1;
-      } else if (num > KMP_INT_MAX) {
```

Comment:
LGTM, but a question.  Why do we still need a KMP_INT_MAX? Can we just use INT_MAX everywhere? Note that __kmp_str_to_int will max out to INT_MAX.  Either we don't need KMP_INT_MAX, or we need to account for the discrepency. We
#define KMP_INT_MAX ((kmp_int32)0x7FFFFFFF).  I assume this was because we wanted it to be a constant that was the same on all architectures.  Is it still needed for anything?    

We now only use it here:
src/kmp_settings.cpp:  __kmp_stg_parse_int(name, value, 0, KMP_INT_MAX, &__kmp_num_locks_in_block);

It looks to me like that could be changed to INT_MAX, and the #define KMP_INT_MAX can be removed.

Reply:
Good point! I'll follow up with removing `KMP_INT_MAX`. It looks like `KMP_INT_MIN` is already unused as well so both can be removed.